assets/i18n/en.json

```diff
@@ -621,15 +621,17 @@
     "I18N_SAVE_PROGRESS": "Log in or sign up to save your progress and play through the next lesson.",
     "I18N_SHARE_LESSON": "Share this lesson",
     "I18N_SHOW_SOLUTION_BUTTON": "Show Solution",
-    "I18N_SIDEBAR_ABOUT_LINK": "About Oppia",
```

Comment:
we will have to delete the key from all the languages, make a search for the key name and please delete from all otherwise backend test will fail.

Reply:
Done!